x-pack/plugin/mapper-aggregate-metric/src/main/java/org/elasticsearch/xpack/aggregatemetric/mapper/AggregateDoubleMetricFieldMapper.java

```diff
@@ -292,6 +296,7 @@ private NumberFieldMapper.NumberFieldType delegateFieldType(Metric metric) {
 
         /**
          * Return a delegate field type for the default metric sub-field
+         *
```

Comment:
Does your formatter do this? I think it's more standard, but I don't tend to do it myself.

Reply:
If I undertand the question well, this has nothing to do with formatting the field in the output. 

This has to do with the following: 

1. Querying the field: When running a `term` or a `range` query, the query is delegated to the subfield metric that has been marked as `default_metric` in the field mapping.
2. When retrieving the field using the fields api, the value of the `default_metric` is returned (this will change when #88534 is merged)

Reply:
Sorry, no, my question was about the extra line inserted into the javadoc.

Reply:
Sorry! I wasn't clear.

Reply:
Looks like it was me messing with IntelliJ formatting. Reverted it now.